solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -1259,6 +1324,47 @@ contract StandardPoolConverter is ConverterVersion, IConverter, ContractRegistry
         return _amount.mul(_reserveBalance) / _supply;
     }
 
+    /**
+     * @dev returns two samples required for calculating the network fee
+     *
+     * @param currProd  the current product of the balances
+     * @param prevProd  the previous product of the balances
+     *
+     * @return the square root of the current product of the balances
+     * @return the square root of the previous product of the balances
+     */
+    function networkFeePoints(
```

Comment:
I'd rather have a more self explanatory name here

Reply:
please suggest

Reply:
I don't quite understand the math step here in order to propose a new name

Reply:
This function returns the geometric mean of each point in the network fee mechanism (the current point, and the previous point recorded, where 'point' stands for the status of the balances (i.e., the balance of the pool in each reserve).

The "problem" here is that this function doesn't receive the actual points (current and previous), but the products, i.e., the product of the current point and the product of the previous point.

So all this function does, really, is to return the square-roots of 2 values.
Unfortunately, there isn't a real good name for it.
Essentially, all this function does is to aggregate a piece of common code which appears in several different places.

The 'networkFee' prefix isn't event relevant here TBH.

We may as well just call it 'squareRoots' or something like that.

Reply:
Removed in https://github.com/bancorprotocol/contracts-solidity/pull/568.

Reply:
Done